src/index.ts

```diff
@@ -31,7 +31,7 @@ export { Team } from './teams/types';
 export { TimesheetAccounting } from './timesheetAccountings/types';
 export { TimeTracking, TimeTrackingCreate, StartTimeTrackingData, StopTimeTrackingData } from './timetrackings/types';
 export { User } from './users/types';
-export { UserStatusOverview } from './userStatusOverview/types';
+export { UserStatusOverview, UserStatusOverviewStatus } from './userStatusOverview/types';
```

Comment:
Can we replace this with export * from ./userStatus... ?

Reply:
will change this